src/bindings/python/tests_compatibility/test_onnx/test_backend.py

```diff
@@ -313,7 +307,8 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
     ),
     (
         skip_dynamic_model,
-        "OnnxBackendNodeModelTest.test_triu_one_row_cpu"
+        "OnnxBackendNodeModelTest.test_triu_one_row_cpu",
+        "OnnxBackendNodeModelTest.test_squeeze_negative_axes_cpu",
```

Comment:
This test is skipped only for compatibility (legacy) api, because unsupported dynamic shapes:
https://github.com/openvinotoolkit/openvino/blob/4f2a041a4de2edc70fef920ab68b99fd968230fc/src/bindings/python/tests_compatibility/__init__.py#L167